apps/processing/src/services/processing.service.ts

```diff
@@ -65,7 +73,12 @@ export class ProcessingService {
         });
 
         try {
-            await this.orchestrator.run(abortController.signal);
+            for (const orchestrator of this.orchestrators.values()) {
+                this.logger.info(`Starting orchestrator for chain ${orchestrator.chainId}...`);
+                orchestratorProcesses.push(orchestrator.run(abortController.signal));
+            }
```

Comment:
```suggestion
            const orchestratorProcesses = this.orchestrators.values().map(orchestrator=>orchestrator.run);
```

Reply:
but we're not passing the AbortController here, the AbortController is so we can gracefully exit the app so it finishes the current event being processed and then exits the app

Reply:
it's missing in my suggestion but you can xd